lib/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "lib"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+name = "lib"
+path = "src/lib.rs"
+
+[dependencies]
+utils = { path = "../utils"}
+kernel = { path = "../kernel" }
+ata = { path = "../drivers/ata" }
+bitflags = "1.3"
+
+[dependencies.lazy_static]
```

Comment:
can you change the dependency to how it is everywhere else, i like that notation better

Reply:
Sure

Reply:
Done